src/main/java/org/opentripplanner/model/projectinfo/MavenVersionParser.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.model.projectinfo;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.InputStream;
+import java.util.Properties;
+
+class MavenVersionParser {
+  private static final Logger LOG = LoggerFactory.getLogger(ProjectInfo.class);
+  private static final String FILENAME = "maven-version.properties";
```

Comment:
Rename "maven-version" to "otp-project-info.properties" and this class to reflect the file name.